app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -268,7 +269,11 @@ class NavigationOverlayFragment : Fragment() {
                 FxaRepo.AccountState.AUTHENTICATED_WITH_PROFILE -> {
                     fxaRepo.accountManager.accountProfile()?.let { profile ->
                         if (profile.avatar != null) {
-                            // TODO: need to fetch avatar
+                            profile.avatar?.url?.let { url ->
+                                PicassoWrapper.client
+                                    .load(url)
```

Comment:
We need to define an error case: what if we're unable to fetch the icon from the server?